08_canvas/index.js

```diff
@@ -0,0 +1,58 @@
+window.addEventListener("load", () => {
+    console.log("hello");
+});
+
+const canvas = document.querySelector("#canvas");
```

Comment:
canvas is a very powerful browser feature for interactive stuff. [threejs](https://threejs.org/) and [D3](https://d3js.org/) are some of the many libraries to help us doing that

It's on my todo list to master it for years now but the right opportunity didn't come yet.

Reply:
Yes its a lot to be honest! I followed tutorial for it...